src/python/pants/ivy/ivy_subsystem.py

```diff
@@ -42,6 +42,8 @@ def register_options(cls, register):
              help='Directory to store artifacts retrieved by Ivy.')
     register('--ivy-settings', advanced=True,
              help='Location of XML configuration file for Ivy settings.')
+    register('--bootstrap-ivy-settings', advanced=True,
```

Comment:
I'm wondering if we should do something to default the value to --ivy-settings if you haven't set this field. I don't want to maintain two copies of ivy-settings.xml if I don't have to.